website/project/model.py

```diff
@@ -736,9 +115,10 @@ def validate_title(value):
 
 
 def validate_user(value):
```

Comment:
Wouldn't it make more sense to put them in `osf/models/validators.py`, given that they... validate?